ptvsd/wrapper.py

```diff
@@ -1224,6 +1215,33 @@ def _process_debug_options(self, opts):
         else:
             redirect_output = ''
         self.pydevd_request(pydevd_comm.CMD_REDIRECT_OUTPUT, redirect_output)
+        self._apply_code_stepping_settings()
+
+    def _is_just_my_code_stepping_enabled(self):
+        """Returns true if just-me-code stepping is enabled.
+
+        Note: for now we consider DEBUG_STDLIB = False as just-my-code.
+        """
+        dbg_stdlib = self.debug_options.get('DEBUG_STDLIB', False)
+        return not dbg_stdlib
+
+    def _apply_code_stepping_settings(self):
+        if self._is_just_my_code_stepping_enabled():
+            project_dirs = []
+            for path in sys.path + [os.getcwd()]:
```

Comment:
I would change the condition to as follows:
```python
if path.endswith('ptvsd'):
    is_stdlib = True
else:
    for prefix in STDLIB_PATH_PREFIXES:
        if norm_path.startswith(prefix):
            is_stdlib = True
            break
```

Reply:
will do